yew/src/html/scope.rs

```diff
@@ -253,40 +259,88 @@ impl<COMP: Component> Scope<COMP> {
     /// Creates a `Callback` which will send a batch of messages back
     /// to the linked component's update method when invoked.
     ///
+    /// The callback function's return type is generic to allow for dealing with both
+    /// Options and Vecs nicely. Option can be used when dealing with a callback that
```

Comment:
Make it clear that we're talking about the types:
```suggestion
    /// `Option` and `Vec` nicely. `Option` can be used when dealing with a callback that
```